kubernetes_platform/python/RELEASE.md

```diff
@@ -9,7 +9,7 @@ Some steps require elevated permissions to push branches, publish the package, a
 
     **Create and merge the PR into the `master` branch.**
 
-2.  [Requires repo OWNER permissions]  Replace the `KFP_KUBERNETES_VERSION` value with the version in
+1.  [Requires repo OWNER permissions]  Replace the `KFP_KUBERNETES_VERSION` value with the version in
```

Comment:
nitpick: It looks like the numbered list here and below accidentally updated (each item is Step 1)

Reply:
ah, auto-increments doing it this way.  TIL.